.github/workflows/main.yaml

```diff
@@ -18,18 +18,18 @@ jobs:
         node-version: [14.x, 16.x, 18.x, 20.x]
 
     steps:
-      - uses: actions/checkout@9bb56186c3b09b4f86b1c65136769dd318469633 # v4.1.2
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
```

Comment:
We're downgrading the action here

Reply:
Since 4.1.2 was pre-release, I've downgraded back to 4.1.1 across all the SDKs to the latest stable version.

Reply:
* https://github.com/openfga/dotnet-sdk/blob/main/.github/workflows/main.yaml#L14
* https://github.com/openfga/python-sdk/blob/main/.github/workflows/main.yaml#L25
* Java SDK (quite old) https://github.com/openfga/java-sdk/blob/main/.github/workflows/main.yml#L40

Reply:
You had pointed out that it was pre-release on a later commit, after those were merged. Forgot which SDK it was offhand, but I updated when you mentioned it. The SDK Generator PR has the correct versions, just need to cut new PRs to downgrade those repos as well. 😄 

Java has a pending PR for it's code coverage that bumps it. https://github.com/openfga/java-sdk/pull/71